mailbox/store/src/main/java/org/apache/james/mailbox/store/event/DefaultDelegatingMailboxListener.java

```diff
@@ -107,4 +107,13 @@ protected void deliverEventToGlobalListeners(Event event) {
         }
     }
 
+    @Override
+    public boolean isListeningTo(MailboxListener mailboxListener) {
```

Comment:
It is a class notified after event creation, that takes care of notifying other listeners about events.